<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>655</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1084</width>
    <height>628</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1084</width>
    <height>655</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Painter</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MyCanvas" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>1041</width>
      <height>536</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
   </widget>
   <widget class="QPushButton" name="openAddingBTN">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Manually add</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>5</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Add Rule</string>
    </property>
    <widget class="QComboBox" name="ruleBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>206</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Distance from point to point</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Distance from point to line</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Three points on line</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ratio between three points</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Angle</string>
      </property>
     </item>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>15</y>
       <width>181</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="ruleValueLabel">
        <property name="text">
         <string>Value: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ruleValueEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QListWidget" name="objectsList">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>10</y>
      <width>46</width>
      <height>46</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="concatinateBTN">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Create segment</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>600</y>
      <width>409</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item alignment="Qt::AlignLeft">
      <widget class="QLabel" name="statusBar">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="messageBar">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>65</y>
      <width>31</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="selectBTNTool">
       <property name="maximumSize">
        <size>
         <width>29</width>
         <height>28</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select (Ctrl + 1)</string>
       </property>
       <property name="text">
        <string>S</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+1</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pointBTNTool">
       <property name="maximumSize">
        <size>
         <width>29</width>
         <height>29</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Draw point (Ctrl + 2)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/pointbtn.png</normaloff>icons/pointbtn.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+2</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ZoomBTNTool">
       <property name="maximumSize">
        <size>
         <width>29</width>
         <height>28</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Zoom+ (Ctrl + 5)</string>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+5</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1084</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionRedraw_all"/>
    <addaction name="actionClear_all"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionRedraw_all">
   <property name="text">
    <string>Redraw all</string>
   </property>
  </action>
  <action name="actionClear_all">
   <property name="text">
    <string>Clear all</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyCanvas</class>
   <extends>QGraphicsView</extends>
   <header>mycanvas.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
